apps/drupal/templates/layout/page.html.twig

```diff
@@ -44,6 +45,9 @@
  */
 #}
 
+{% if not components %}
+  {% include '@particle/layout/components--error-message.html.twig' %}
```

Comment:
This `@particle` namespace: is this Drupal core providing the path to the theme?

Reply:
yes, this is a drupal standard